README-sycl.md

```diff
@@ -260,15 +277,21 @@ Output (example):
 [opencl:cpu:1] Intel(R) OpenCL, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz OpenCL 3.0 (Build 0) [2023.16.10.0.17_160000]
 [opencl:gpu:2] Intel(R) OpenCL Graphics, Intel(R) Iris(R) Xe Graphics OpenCL 3.0 NEO  [31.0.101.5186]
 [ext_oneapi_level_zero:gpu:0] Intel(R) Level-Zero, Intel(R) Iris(R) Xe Graphics 1.3 [1.3.28044]
-
 ```
 
 3. Install cmake & make
 
-a. Download & install cmake for windows: https://cmake.org/download/
+a. Download & install cmake for Windows: https://cmake.org/download/
+
+b. Download & install make for Windows provided by mingw-w64
+
+- Download binary package for Windows in https://github.com/niXman/mingw-builds-binaries/releases.
```

Comment:
VS build is more common for Windows users. Just a thought.

Reply:
Yes. But I meet this issue in a new PC.
I think it's better record here.

Reply:
We recommend w64devkit for MinGW in the main README, I don't know why we would recommend yet another MinGW distribution here.

Reply:
Here use the make.exe in MinGW only.
w64devkit is docker env.
In the chapter **Windows**, we provide the native build method. So not use w64devkit.
In the chapter **Docker**, there is docker method.

I will find another method to install make.exe without MinGW.

Thank you!

Reply:
@cebtenzzre Update as your suggestion: https://github.com/ggerganov/llama.cpp/pull/5297